docs/reference/commandline/run.md

```diff
@@ -220,7 +220,17 @@ An example of a file passed with `--env-file`
     TEST_APP_DEST_PORT=8888
     TEST_PASSTHROUGH=howdy
 
-    $ docker run --name console -t -i ubuntu bash
+    $ docker run --env-file ./env.list busybox env
+    HOME=/
+    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    HOSTNAME=5198e0745561
+    TEST_FOO=BAR
+    TEST_APP_DEST_HOST=10.10.0.127
+    TEST_APP_DEST_PORT=8888
+    TEST_PASSTHROUGH=
```

Comment:
Oh yeah, good idea to add more example of valid environment variable, I'll add some :wink: